src/thread.rs

```diff
@@ -208,6 +208,9 @@ pub struct ScopedJoinHandle<'a, T: 'a> {
     _marker: PhantomData<&'a T>,
 }
 
+unsafe impl<'a, T> Send for ScopedJoinHandle<'a, T> {}
+unsafe impl<'a, T> Sync for ScopedJoinHandle<'a, T> {}
```

Comment:
I've also submitted a PR that adds similar impls for `JoinHandle`:
https://github.com/rust-lang/rust/pull/52759